test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -9,7 +9,7 @@
 import org.junit.Test;
 
 public class UtilsTest {
```

Comment:
Hi Daniel! I suppose this update is for another LO?

Reply:
Hi Yes, thank you for reminding me :) I did try my best to make a "clean" PR but could not do so :(

Unfortunately some unrelated commits appeared here. Not sure why... Because last time I resolved some conversations with @liliwei25 in the previous PR for W4.11e by making some commits.